src/components/subscriptions/licenses/LicenseManagementModals/LicenseManagementRemindModal.jsx

```diff
@@ -112,7 +106,13 @@ const LicenseManagementRemindModal = ({
       if (userEmailsToRemind.length > 0) {
         options.user_emails = userEmailsToRemind;
       } else {
-        options.filters = transformedFilters;
+        // If the UI happened to render bulk actions without any state set for the table or just a filter is set
+        // with no selected items.
+        logError(`Unable to remind license(s) based on table state,
+        remindAllUsers: ${remindAllUsers},
+        userEmailsToRevoke: ${userEmailsToRemind},
+        filters: ${activeFilters}`);
```

Comment:
fix weird indentation.  Honestly I might just simplify this to just "Unable to remind license(s) based on table state.  No licenses selected for reminder."